docs/docs/guides/multitrack-progress.md

```diff
@@ -94,12 +94,11 @@ import { useProgressStateStore } from '../store';
 const setProgress = useProgressStateStore.getState().setProgress;
 
 export const PlaybackService = async function() {
-  TrackPlayer.addEventListener(Event.PlaybackProgressUpdated, async () => {
+  TrackPlayer.addEventListener(Event.PlaybackProgressUpdated, async (event) => {
     // get the position and currently playing track.
-    const position = TrackPlayer.getPosition();
-    const track = TrackPlayer.getCurrentTrack();
+    const track = await TrackPlayer.getActiveTrack();
```

Comment:
We don't want to just get the active track here, as it's possible it's unloaded by this point. We should fetch it based on the ID in the event payload. I'd recommend something like my original comment.